backend/compact-connect/stacks/persistent_stack/bulk_uploads_bucket.py

```diff
@@ -145,6 +146,8 @@ def _add_ingest_object_events(self, event_bus: EventBus):
             entry=os.path.join('lambdas', 'license-data'),
             index=os.path.join('handlers', 'bulk_upload.py'),
             handler='process_s3_event',
+            timeout=Duration.minutes(15),
+            memory_size=1024,
```

Comment:
is this the amount of memory allocated for the bulk upload job? or something else?

Reply:
This is for the lambda that reads/parses the bulk-upload file that was just uploaded to s3. It doesn't actually need the memory but AWS also scales compute and network bandwidth with your provisioned memory size. This is actually allowing us to download the file from s3 faster.